embed_grader/serapis/templates/serapis/assignment.html

```diff
@@ -66,6 +66,10 @@ <h2 class="page-header">Assignment - {{ assignment.name }}</h2>
                     <a href="{% url 'zip-input-files' task.id %}" class="btn btn-primary"
                         style="font-size:12px; background:white; color:SteelBlue; border-color:SteelBlue;">
                       <span class="glyphicon glyphicon-download-alt"></span> &nbsp;Download
+                    </a>&nbsp;
+                    <a href="{% url 'view-task-input-files' task.id %}" class="btn btn-primary"
+                        style="font-size:12px; background:white; color:SteelBlue; border-color:SteelBlue;">
```

Comment:
Can we put `font-size:12px; background:white; color:SteelBlue; border-color:SteelBlue;` into a class, too? I feel whenever we have a button, we always apply the same css

Reply:
Or another idea is that we can design an `add_button` tag, with input span_symbol_class, text, and url params!

Reply:
hmm... since the button already used the bootstrap "btn btn-primary" class, I don't think we can add additional css in this class? And also I don't think replacing the style with an add_button tag will simplify the code, because we still need to pass in many params. What do you think?

Reply:
I think having an `add_button` tag will simply codes, right now whenever we want to add a button, we need to write at least 3 lines, with a lot of repeating messages like the classes to be applied, the desired styles, and so on. These can all go into the `add_button` tag (was thinking `show_button` may be a better name ha). However, I think at minimum we need to create a new class (let's call it `show-button`), and move `"font-size:12px; background:white; color:SteelBlue; border-color:SteelBlue;"` into this class.